client/src/components/Feed/FilterBox.js

```diff
@@ -3,35 +3,40 @@ import styled from "styled-components";
 import { Modal, List } from "antd-mobile";
 import CustomList from "../../components/List/CustomList";
 import CustomButton from "../../components/Button/CustomButton";
-import FilterTitle from "../Typography/Title/FilterTitle";
-import FilterOptionButton from "../Button/FilterOptionButton";
+import CustomH1 from "../Typography/Title/CustomH1";
+import DownArrowButton from "../Button/DownArrowButton";
 import FilterAccordion from "./FilterAccordion";
 import { FeedContext } from "../../pages/Feed";
+import { ROYAL_BLUE, SELAGO, DARK_GRAY } from "../../constants/colors";
 
 const FilterBoxWrapper = styled.div`
   margin-bottom: 4rem;
 `;
 
 export default () => {
   const feedContext = useContext(FeedContext);
-  const { filters, modal, handleModal, handleQuit } = feedContext;
+  const { filters, filterModal, handleFilterModal, handleQuit } = feedContext;
   const renderFilterOptions = (filters) => {
     return filters.map((filter, idx) => (
-      <FilterOptionButton
+      <DownArrowButton
         key={idx}
         label={filter.label}
-        handleClick={handleModal(idx)}
+        handleClick={handleFilterModal(idx)}
```

Comment:
no need for `() => ` here? or it will assign the return value of `handleFilterModal()` as handler?

Reply:
Ah it might not be clear lookong from here but `handleFilterModal()` returns a function